guides/mobile_app_guidance.md

```diff
@@ -0,0 +1,46 @@
+Mobile Application Guidance
+===========================
+
+Scope
+-----
+
+This guidance applies to mobile application in-house development and management.
+
+This guidance is intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+This document should be read in conjunction with the Mobile Application Standards.
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
```

Comment:
Nitpick, GDS style guide states DEFRA should be written as "Defra"

Reply:
OK